packages/ethereum-contracts/test/contracts/agreements/InstantDistributionAgreementV1.test.js

```diff
@@ -309,6 +310,48 @@ describe("Using InstantDistributionAgreement v1", function () {
                     "IDA: E_LOW_BALANCE"
                 );
             });
+
+            it("#1.1.9 publisher should not be able to update subscription to zero address", async () => {
+                await shouldCreateIndex({
+                    testenv: t,
+                    superToken,
+                    publisherName: "alice",
+                    indexId: DEFAULT_INDEX_ID,
+                });
+
+                await expectRevert(
+                    shouldUpdateSubscription({
+                        testenv: t,
+                        superToken,
+                        publisherName: "alice",
+                        indexId: DEFAULT_INDEX_ID,
+                        subscriberAddress: ZERO_ADDRESS,
```

Comment:
since subscriberAddress to ZERO_ADDRESS is only for expectRevert test cases, we don't need to modify the behaviour test code. I would say in these case just do the call directly through the js-sdk wrapper and expect the revert.